deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -293,11 +270,12 @@ export class MigrationConsoleStack extends MigrationServiceCore {
             "MIGRATION_SOLUTION_VERSION": props.migrationsSolutionVersion,
             "MIGRATION_SERVICES_YAML_PARAMETER": parameter.parameterName,
             "MIGRATION_SERVICES_YAML_HASH": hashStringSHA256(servicesYaml.stringify()),
+            "SHARED_LOGS_DIR_PATH": `${sharedLogFileSystem.mountPointPath}/migration-console-${props.defaultDeployId}`,
```

Comment:
Does anything in the migration console use this currently? I'm guessing the tools like MetadataMigration will?

Reply:
Not currently, but I was aiming for consistency with our 'core' containers.  Yup - I plan for metadata migration to be able to log here if need be.